### PR TITLE
[Arista] add MMU configuration for Arista 7260 C64

### PR DESCRIPTION
**Pull request by yxieca**
#### How I did it
Add MMU configuration for Arista 7260 C64 hardware sku

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

**Review by neethajohn — APPROVED** [review]

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/th2-a7260cx3-64-64x100G.config.bcm [hunk 1]
```diff
@@ -1011,3 +1011,5 @@ serdes_preemphasis_114=0x174507
 serdes_preemphasis_115=0x184606
 serdes_preemphasis_116=0x103706
 serdes_preemphasis_117=0x133c06
+
+mmu_init_config="MSFT-TH2-Tier1"
```
Comment (lguohan) [inline device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/th2-a7260cx3-64-64x100G.config.bcm:1015]: C64 profile can also be Tier0, how to handle that?

Comment (yxieca) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/th2-a7260cx3-64-64x100G.config.bcm]: Good catch. Added T0 config.

**Review by lguohan — APPROVED** [review]

**Pull request merged**